osu.Game.Tests/Visual/Multiplayer/TestSceneMultiplayerParticipantsList.cs

```diff
@@ -198,7 +198,7 @@ public void TestKickButtonOnlyPresentWhenHost()
 
             AddStep("make second user host", () => MultiplayerClient.TransferHost(3));
 
-            AddUntilStep("kick buttons not visible", () => this.ChildrenOfType<ParticipantPanel.KickButton>().Count(d => d.IsPresent) == 0);
+            AddUntilStep("kick buttons not visible", () => !this.ChildrenOfType<ParticipantPanel.KickButton>().Any(d => d.IsPresent));
```

Comment:
The story about `Any` vs `Count` is beyond this. CA1860 is about using `.Count > 0` instead of predicate-less `.Any()`, and has a lot of violations in both game and test code. When using a predicate, the performance characteristic reverses.

> To the point that rider has begun flagging `.Any()` usages and suggesting to replace them with count checks at hint level.

Do you mean predicate-less `Any`? This is the same as CA1860.

To be straight, performance-wise, `.Count` is better than `.Any()`, but `Any()` is better than `.Count()`, and `.Any(predicate)` is better than `.Count(predicate)`. This leads to inconsistent style for maximizing performance.

I'm very sure that the performance is not important here. The question is whether to prefer `.Count > 0` over `Any()` for every cases - it has about 30 violations in game code base.

Reply:
> The question is whether to prefer `.Count > 0` over `Any()` for every cases

I'd say it's situational, at least for test code. At the end of the day it's up to the reviewers to be able to understand the code and feel whether it's maintainable or not.

For operational code, I think we can do manual review to keep things simple.

Personally, whenever I see `!x.Any(y)`, I always have to go and check the documentation for the empty case especially as it relates to `x.All(!y)`. With this context, `Count()` is easier to mentally parse for me.

Reply:
Meets my expectation to keep things as-is and review in future.